app/Console/Commands/SendActiveMembersEmail.php

```diff
@@ -44,13 +44,17 @@ public function handle()
         $users = \App\Models\User::where('status', 'active')->orderby('first_preferred')->orderby('last_preferred')->get();
 
         // find authentications from the past month, if more than 3 add them to the list
-        $email_data['member_list'] = [];
-        foreach ($users as $user) {
-            $recs = \App\Models\Authentication::where('user_id', $user['id'])->where('gatekeeper_id', '11')->where('created_at', '>=', Carbon::now()->subMonth()->toDateTimeString())->get();
-            if (count($recs) >= 3) {
-                $email_data['member_list'][] = $user['first_preferred'].' '.$user['last_preferred'];
-            }
-        }
+        $email_data['member_list'] = \App\Models\User::where('status', 'active')
+            ->whereHas('authentications', function ($query) {
+                $query->whereRelation('gatekeeper', 'type', 'doorway')
+                    ->where('lock_in', '>=', Carbon::now()->subMonth()->toDateTimeString());
```

Comment:
Not sure whether `lock_in` or `created_at` values are the same, similar, or at all comparable. I'll have to review how they're composed before confirming.